### PR TITLE
feat(About): add ability to use product title image

### LINKED ISSUE #197: About should allow for an optional image for product name
Enhance the About screen to optionally take an image (which is text) which can be shown on the About screen.

This is a requirement for some of our products don't have product identity logos, but have an image including the product name in a specific font.  

Note that this is already supported in login & masthead ( top banner )

### PR DESCRIPTION
**Pull request by jeff-phillips-18**
**What**:
Adds the ability to pass an image (or other node) for the product title in the about modal.

**Link to Storybook**:
https://jeff-phillips-18.github.io/patternfly-react/

**Additional issues**:
Fixes #197

### CODE REVIEW

File: src/components/AboutModal/AboutModal.js [hunk 1]
```diff
@@ -51,8 +51,8 @@ AboutModal.propTypes = {
   show: PropTypes.bool.isRequired,
   /** Function to call when modal is closed */
   onHide: PropTypes.func.isRequired,
-  /** Text to show for the product title */
-  productTitle: PropTypes.string,
+  /** Text or Element to show for the product title */
+  productTitle: PropTypes.oneOfType([PropTypes.string, PropTypes.node]),
```
Comment (sharvit) [inline src/components/AboutModal/AboutModal.js]: I think it can be as simple as:
```jsx
productTitle: PropTypes.node,
```

Since `node` can be a `string`.

Comment (jeff-phillips-18) [reply src/components/AboutModal/AboutModal.js]: Thanks. Updated.

**Comment by priley86** [comment]
+1... for `fix()` ... and happy to see it merge after...

**Comment by priley86** [comment]
validated in my semantic-release playground that `fix()` and `perf()` do a patch level release, and that `refactor()` and `revert()` does no release. `feat()` will do a minor release.

**Comment by cdcabrera** [comment]
@priley86 @jeff-phillips-18  

For the most part we tend to keep our use of semantic release types limited. Going to go ahead and look like I'm flip-flopping, and merge this in. 

There is a bit of subjective to these, and this is one of those in-between updates that's kind of enhancing the existing component and not breaking previous behavior. Where fix seems like an odd choice, since the component isn't actually broken. For future reference, based on the Commitizen types, per @priley86 ```perf``` and, of course, ```fix``` appear to increment as patches.

**Pull request merged**